### DISCUSSION TITLE
DNSSEC must be disabled

### DISCUSSION THREAD

**Discussion by cswkim** [vercel/community#5597] [Help] [answered]
### Summary

After transferring to a new domain provider from Google Domains and using Vercel's nameservers I get an error indicating:

`Your domain's nameservers are pointing to Vercel, but you must disable DNSSEC with your domain registrar in order for your nameservers to resolve globally.`

DNSSEC is already disabled with my new registrar.

### Steps to Reproduce

![Screen Shot 2024-01-25 at 1 11 55 AM](https://github.com/vercel/community/assets/1006807/a57dddaf-e76c-4dc9-8d0d-2884510a07c4)

**Comment by cswkim** [comment, marked as answer]
Nevermind, I just switched to using A and CNAME records and everything seems to be working now.

**Reply by zwergdev** [reply]
@cswkim Hi, you have purchased a domain on Vercel?
